kt/godot-library/src/main/kotlin/godot/tests.kt

```diff
@@ -15,28 +15,37 @@ open class Object(isRef: Boolean = false) : KtObject(isRef) {
     }
 
     fun connect(name: String, target: Object, method: String, binds: VariantArray, flags: Long): GodotError {
-        val refreshBuffer = TransferContext.writeArguments(KtVariant(name), KtVariant(target), KtVariant(method), KtVariant(binds), KtVariant(flags))
-        TransferContext.callMethod(rawPtr, OBJECT, OBJECT_CONNECT, KtVariant.Type.LONG, refreshBuffer)
-        return GodotError.values()[TransferContext.readReturnValue().asInt()]
+        val refreshBuffer = TransferContext.writeArguments(
```

Comment:
Oh and: Why do these local variables exist?
They are unused and of type `Unit`

Reply:
legacy, forgot to change this